src/options.json

```diff
@@ -0,0 +1,72 @@
+{
+  "type": "object",
+  "properties": {
+    "attrs": {
+      "anyOf": [{ "type": "array" }, { "type": "string" }]
+    },
+    "minimize": {
+      "type": "boolean"
+    },
+    "removeComments": {
+      "type": "boolean"
+    },
+    "removeCommentsFromCDATA": {
+      "type": "boolean"
+    },
+    "removeCDATASectionsFromCDATA": {
+      "type": "boolean"
+    },
+    "collapseWhitespace": {
+      "type": "boolean"
+    },
+    "conservativeCollapse": {
+      "type": "boolean"
+    },
+    "removeAttributeQuotes": {
+      "type": "boolean"
+    },
+    "useShortDoctype": {
+      "type": "boolean"
+    },
+    "keepClosingSlash": {
+      "type": "boolean"
+    },
+    "minifyJS": {
+      "type": "boolean"
+    },
+    "minifyCSS": {
+      "type": "boolean"
+    },
+    "removeScriptTypeAttributes": {
+      "type": "boolean"
+    },
+    "removeStyleTypeAttributes": {
+      "type": "boolean"
+    },
+    "root": {
+      "type": "string"
+    },
+    "interpolate": {
+      "anyOf": [{ "type": "boolean" }, { "type": "string" }]
+    },
+    "exportAsDefault": {
+      "type": "boolean"
+    },
+    "exportAsEs6Default": {
+      "type": "boolean"
+    },
+    "ignoreCustomFragments": {
+      "type": "array",
+      "items": {
+        "instanceof": "RegExp"
+      }
+    },
+    "config": {
+      "type": "string"
+    },
+    "caseSensitive": {
+      "type": "boolean"
+    }
+  },
+  "additionalProperties": false
```

Comment:
Let's move all `minimizer`'s options under the one option like:
`minimize: boolean || Object`